src/schemathesis/cli/validators.py

```diff
@@ -13,10 +14,12 @@ def validate_schema(ctx: click.core.Context, param: click.core.Option, raw_value
 
 def validate_auth(
     ctx: click.core.Context, param: click.core.Option, raw_value: Optional[str]
-) -> Optional[Tuple[str, str]]:
+) -> Optional[Union[Tuple[str, str], HTTPDigestAuth]]:
```

Comment:
Maybe we can use `Optional[AuthBase]` and always wrap that tuple into `HTTPBasicAuth` ? types will be simpler